server/etcdmain/config.go

```diff
@@ -186,11 +186,25 @@ func newConfig() *config {
 		"advertise-client-urls",
 		"List of this member's client URLs to advertise to the public.",
 	)
+	fs.BoolVar(&cfg.ec.EnableV2Discovery, "enable-v2-discovery", cfg.ec.EnableV2Discovery, "Enable to bootstrap the cluster using v2 discovery. Will be deprecated in v3.7, and be decommissioned in v3.8.")
```

Comment:
Why not deprecate in v3.6?

Reply:
You mentioned in [issues/13624](https://github.com/etcd-io/etcd/issues/13624) that v3 discovery will be marked as default in 3.7 and the v2 discovery will be removed in 3.8.  My understanding is that if v2 discovery is deprecated in 3.6, then v3 discovery should be marked as default in 3.6 instead of 3.7? Did I miss anything?

```
Deprecation plan:

    Make v3 default in v3.7
    Remove v2 option in v3.8

```

Reply:
ok, you're right.